packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,173 @@
+/* @license CC0-1.0 */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Textarea } from '@utrecht/component-library-react/src/css-module/Textarea';
+import readme from '@utrecht/components/textarea/README.md?raw';
+import tokensDefinition from '@utrecht/components/textarea/tokens.json';
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import React from 'react';
+import { designTokenStory } from './util';
+
+const defaultArgs = {
+  dir: 'auto',
+  disabled: false,
+  focus: false,
+  hover: false,
+  invalid: false,
+  placeholder: undefined,
```

Comment:
Placeholder used to be `placeholder: ''`, not it is `undefined`. This affects the args table, and makes it impossible to experiment with this property.

## Main branch

<img width="781" alt="Screenshot 2023-08-29 at 08 59 46" src="https://github.com/nl-design-system/utrecht/assets/30694/703c7bf2-3292-4e5c-a128-360d7ba80203">

## This branch

<img width="704" alt="Screenshot 2023-08-29 at 08 59 31" src="https://github.com/nl-design-system/utrecht/assets/30694/acdf30a4-295a-4a5e-9631-085f958aadd7">

...and after clicking the button...

<img width="698" alt="Screenshot 2023-08-29 at 09 00 31" src="https://github.com/nl-design-system/utrecht/assets/30694/c4b49cef-9857-4746-b5f2-276102e7d9b2">